styx-common/src/test/java/com/spotify/styx/util/TimeUtilTest.java

```diff
@@ -244,29 +245,44 @@ public void shouldReturnEmptyList() {
         instantsInRange(firstTimeHours, lastTimeHours, Schedule.parse("0 * * * *")).isEmpty());
   }
 
-  @Test(expected = IllegalArgumentException.class)
+  @Test
   public void shouldGetExceptionIfLastInstantIsBeforeFirstInstant() {
     final Instant firstTimeHours = parse("2016-01-19T10:00:00.00Z");
     final Instant lastTimeHours = parse("2016-01-19T09:00:00.00Z");
-    instantsInRange(firstTimeHours, lastTimeHours, Schedule.HOURS);
+    try {
+      instantsInRange(firstTimeHours, lastTimeHours, Schedule.HOURS);
+      fail();
```

Comment:
Use `ExpectedException` rule

Reply:
Thought about this. What is the guideline for using this rule? In case it is quite a bit code achieving the same goal.

Reply:
nvm, it is worth it.